instrumentation/nocode/src/main/java/com/splunk/opentelemetry/instrumentation/nocode/NocodeInitializer.java

```diff
@@ -23,14 +23,27 @@
 import io.opentelemetry.javaagent.tooling.BeforeAgentListener;
 import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
 import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 @AutoService(BeforeAgentListener.class)
 public class NocodeInitializer implements BeforeAgentListener {
+  private static final String NOCODE_YMLFILE = "splunk.otel.instrumentation.nocode.yml.file";
+  private static final Logger logger = Logger.getLogger(NocodeInitializer.class.getName());
 
   @Override
   public void beforeAgent(AutoConfiguredOpenTelemetrySdk autoConfiguredOpenTelemetrySdk) {
     ConfigProperties config = getConfig(autoConfiguredOpenTelemetrySdk);
-    YamlParser yp = new YamlParser(config);
-    NocodeRules.setGlobalRules(yp.getInstrumentationRules());
+    String yamlFileName = config.getString(NOCODE_YMLFILE);
+    List<NocodeRules.Rule> instrumentationRules = Collections.EMPTY_LIST;
```

Comment:
you could avoid the unchecked assignment warning with
```suggestion
    List<NocodeRules.Rule> instrumentationRules = Collections.emptyList();
```